docs/configuration-reference.md

```diff
@@ -2045,6 +2045,22 @@ remote_write:
     [ sending_queue: <otlpexporter.sending_queue> ]
     [ retry_on_failure: <otlpexporter.retry_on_failure> ]
 
+    # automatically log lines to Loki for discovery/metrics
+    automatic_logging:
```

Comment:
Just curious: what enables this? Is it defining `automatic_logging` in the config or is it providing a value for `loki_name`? Can we document which?

Reply:
Adding `automatic_logging` will instantiate the processor, but it will fail with an error message if it can't find the provided `loki_name`.